### PR TITLE
fix errors found by compiling with python3

### PR DESCRIPTION
**Pull request by davidlange6**
Fix remaining python3 compilation errors in FWCore subsystem

most were

Compiling './GuiBrowsers/python/Vispa/Gui/ZoomableScrollArea.py'...
*** Sorry: TabError: inconsistent use of tabs and spaces in indentation (ZoomableScrollArea.py, line 67)

which I fixed by applying autopep8 (which fixes lots of other whitespace in a standard way)

[its probably a bad sign for the future that ~8 files in FWCore have these space/tab issues...]

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: FWCore/ParameterSet/python/Config.py [hunk 1]
```diff
@@ -1024,87 +1220,126 @@ def _pruneModules(self, d, scheduledNames):
...
         self._insertManyInto(adaptor, "@all_aliases", self.aliases_(), True)
         # This will visit all the paths and endpaths that are scheduled to run,
         # as well as the Tasks associated to them and the schedule. It remembers
         # the modules, ESSources, ESProducers, and services it visits.
         nodeVisitor = NodeVisitor()
         self._insertPaths(adaptor, nodeVisitor)
-        all_modules_onTasksOrScheduled = { key:value for key, value in six.iteritems(all_modules) if value in nodeVisitor.modules }
-        self._insertManyInto(adaptor, "@all_modules", all_modules_onTasksOrScheduled, True)
+        all_modules_onTasksOrScheduled = {
+            key: value for key,
```
Comment (Dr15Jones) [inline FWCore/ParameterSet/python/Config.py]: this is an unfortunate line break choice

File: FWCore/ParameterSet/python/MassReplace.py [hunk 2]
```diff
@@ -1,147 +1,228 @@
...
-                                 nested = cms.PSet(src = cms.InputTag("b"), src2 = cms.InputTag("c")),
-                                 nestedv = cms.VPSet(cms.PSet(src = cms.InputTag("b")), cms.PSet(src = cms.InputTag("d"))),
-                                 vec = cms.VInputTag(cms.InputTag("a"), cms.InputTag("b"), cms.InputTag("c"), cms.InputTag("d"))
-                                )
-            p.s = cms.Sequence(p.a*p.b*p.c)
-            massSearchReplaceAnyInputTag(p.s, cms.InputTag("b"), cms.InputTag("new"))
+                                 nested=cms.PSet(
+                src=cms.InputTag("b"), src2=cms.InputTag("c")),
+                nestedv=cms.VPSet(
+                cms.PSet(
```
Comment (Dr15Jones) [inline FWCore/ParameterSet/python/MassReplace.py]: I'm surprised this did not get indented

File: FWCore/ParameterSet/python/Types.py [hunk 3]
```diff
@@ -415,49 +534,73 @@ def _valueFromString(value):
 
     def pythonValue(self, options=PrintOptions()):
         return str(self.__start) + ', ' + str(self.__startLumi) + ', ' + str(self.__startSub) + ', ' \
-               + str(self.__end)  + ', ' + str(self.__endLumi) + ', ' + str(self.__endSub)
+            + str(self.__end) + ', ' + str(self.__endLumi) + \
+            ', ' + str(self.__endSub)
+
     def cppID(self, parameterSet):
-        return parameterSet.newEventRange(self.start(), self.startLumi(), self.startSub(), self.end(), self.endLumi(), self.endSub())
+        return parameterSet.newEventRange(self.start(), self.startLumi(
```
Comment (Dr15Jones) [inline FWCore/ParameterSet/python/Types.py]: weird line break here

File: FWCore/ParameterSet/python/Types.py [hunk 4]
```diff
@@ -1491,87 +1863,102 @@ def testEventRange(self):
...
+                     k=untracked.int64(7),
+                     l=untracked.uint64(7),
+                     m=untracked.double(7.0),
+                     n=FileInPath("xxx"),
+                     o=untracked.vint32(7, 8),
+                     p=untracked.VInputTag(InputTag("b"), InputTag("c")),
+                     q=untracked.VESInputTag(ESInputTag("c"), ESInputTag("d")),
+                     r=untracked.VEventID(EventID(1, 1, 1), EventID(2, 2, 2)),
+                     s=untracked.VLuminosityBlockID(
+                LuminosityBlockID(
```
Comment (Dr15Jones) [inline FWCore/ParameterSet/python/Types.py]: Weird indenting

File: FWCore/ParameterSet/python/Types.py [hunk 5]
```diff
@@ -1586,28 +1973,33 @@ def testnumbers(self):
...
         def teststring(self):
             self.assertGreater(string("I am a string"), "I am a strinf")
             self.assertGreaterEqual("I am a string", string("I am a string"))
             self.assertLess(5, string("I am a string"))
+
         def testincompatibletypes(self):
             import sys
-            if sys.version_info < (3, 0): #python 2, comparing incompatible types compares the class name
+            if sys.version_info < (
+                    3, 0):  # python 2, comparing incompatible types compares the class name
```
Comment (Dr15Jones) [inline FWCore/ParameterSet/python/Types.py]: poor choice of line break

**Comment by Dr15Jones** [comment]
+1
definitely some arguable choices made by the tool

**Comment by davidlange6** [comment]
let me see if something better can be done with the line breaks..

I think the unit test is failing because I changed whitespace. but to be confirmed.

**Comment by davidlange6** [comment]
will reopen with new approach

**Pull request closed without merging**